src/lib/drivers/driver.js

```diff
@@ -143,8 +143,11 @@ class DriverBase {
         includeCommandLineAPI: true
       });
 
-      // If this gets to 15s and it hasn't been resolved, reject the Promise.
-      asyncTimeout = setTimeout(reject, 15000);
+      // If this gets to 60s and it hasn't been resolved, reject the Promise.
+      asyncTimeout = setTimeout(
+        _ => reject(new Error('aXe did not complete in the allotted time of 60s')),
```

Comment:
The code here is not specific to aXe, changing a global timeout to 60s is risky as well.

It may be worth looking into turning off aXe for certain sites, or pipping a way to configure this timeout on a per test basis.